Source/Core/Core/PatchEngine.cpp

```diff
@@ -156,9 +156,7 @@ void LoadPatches()
   ActionReplay::LoadAndApplyCodes(globalIni, localIni);
 
   // lil silly
```

Comment:
Oh, maybe it's referring to having to use both LoadCodes and SetActiveCodes, not the situation with the out parameter... I dunno. You don't need to remove it if you don't agree with me.